sapphire/simulations/gammas.py

```diff
@@ -60,30 +61,30 @@ def compton_energy_transfer(gamma_energy):
     return compton_edge(gamma_energy) * conversion_factor
 
 
-def dsigma_dT(E, T):
```

Comment:
`dsigma_dt` is totally unrelated to  `dsigma_dT` 

Let's rename it to an english phrase, or keep `T`.